Makefile

```diff
@@ -18,7 +18,6 @@ PY_PATH = $(PYTHONPATH):$(PWD)
 
 .PHONY: bootstrap
 bootstrap:
-	[ "$$VIRTUAL_ENV" != "" ]
```

Comment:
Could we keep it and just set to some fake value in the actions? Having this check is very useful in local development, you don't want to accidentally install stuff globally (but it's fine in CI).